app/controllers/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
   def require_authentication
     if current_person.nil?
       session[:return_to] = request.path
-      redirect_to '/auth/google_apps'
+      redirect_to '/auth/google_oauth2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.